string some_info
---
string ans_str
